### DISCUSSION TITLE
Would like to use this to support non-profit buildings

### DISCUSSION THREAD

**Discussion by ShamanTcler** [mikeberger/borg_calendar#215] [General]
Ok in my case a church, but very applicable to everyone.  We have an old building that needs upkeep ... as in:

- check first aid kits monthly
- change air filters quarterly

Can I set up BORG to send out email reminders as these tasks come do? If so is there a Youtube or other doc I can follow to set this up?

I did check in "task creation" ... I fumbled around and found "BORG" and let me link an address ... for grins and giggles I added an address and a link to a local file. 

Will BORG now send and email with that document as a reminder? (the doc is a list of what to do)

**Comment by mikeberger** [comment]
I'd recommend using google calendar. You can share calendars and send reminder emails with google calendar. You can see your tasks on your phone or any computer and get popup reminders on your phone.

A while ago, I enhanced BORG to sync with google. It turns out that now I just interact with google calendar on my phone 95% of the time rather than starting up BORG. 

BORG can send a reminder email each day with the next day's items and any overdue todos.  It is just a list of items - no attached documents. something like this:
---
Appointments for Jul 13, 2024:
pay water bill
garbage to curb

Past Due:
renew license
---
It's primitive, but was very useful before smartphones existed.
